web/client/plugins/styleeditor/StyleCodeEditor.jsx

```diff
@@ -198,11 +198,11 @@ function StyleEditor({
                     <Toolbar
                         buttons={[
                             {
-                                className: 'square-button-md no-border',
-                                glyph: 'code',
-                                active: editorType === 'textarea',
+                                className: 'square-button-xs no-border',
```

Comment:
Please remove `square-button-xs` because it's not a valid class and we don't need it